features/system_specs/system_specs.feature

```diff
@@ -106,4 +106,4 @@ Feature: System spec
         When I run `rspec spec/system/widget_system_spec.rb`
         Then the output should contain "1 example, 0 failures"
         And the output should not contain "starting Puma"
-        And the exit status should be 0
+        And the exit status should be 0 or I debug
```

Comment:
If you understand why, I am interested. I wrote this https://github.com/rspec/rspec-rails/pull/2325#issuecomment-625967122